starter-kits/credential-registry/server/tob-api/icat_cbs/management/commands/reprocess_credentials.py

```diff
@@ -1,15 +1,10 @@
-from django.core.management.base import BaseCommand, CommandError
-from django.db import transaction, DEFAULT_DB_ALIAS
+from django.core.management.base import BaseCommand
```

Comment:
This script depends on both `icat_cbs` and `tob_api`: the decision to place it in `icat_cbs` was made since I did not want to have a direct diependency to this sub-module in the main app - while it seems acceptable the other way around.

The reason why `SolrQueue` is in `tob_api` is that it is a dependency of the boot script, and I preferred to move it there following the above considerations. I am not against moving things around, if this seems more appropriate and there is a reason to do so.